LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 eLife Sciences Publications, Ltd
+Copyright (c) 2019 Libero
```

Comment:
https://libero.pub/ - has the copyright statement to `eLife Sciences Publications, Ltd`

so which is it @BlueReZZ

Reply:
It's eLife Sciences Publications, Ltd as Libero isn't a legal entity.

Reply:
You can use the one in `community` as the template: https://github.com/libero/community/blob/master/LICENSE

Reply:
> It's eLife Sciences Publications, Ltd as Libero isn't a legal entity.

My bad. I had seen that some of the split repo's had `Libero` as the copyright owner and I some had `eLife Sciences Publications, Ltd`. I will address this for the reviewer repos.